tests/cms/views/interactions/test_vote_action.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from django.contrib.messages.storage.fallback import FallbackStorage
+from django.test import RequestFactory
+
+from integreat_compass.cms.models import User, Vote
+from integreat_compass.cms.views.interactions import VoteFormView
+
+
+def get_request(offer_version_id):
+    request = RequestFactory().post(
+        "", {"approval": "False", "offer_version_id": offer_version_id}
+    )
+    request.user = User.objects.get(pk=1)
+    setattr(request, "session", "session")
+    messages = FallbackStorage(request)
+    setattr(request, "_messages", messages)
+    return request
+
+
+@pytest.mark.django_db
+def test_add_vote_if_not_exists(load_test_data):
+    assert not Vote.objects.filter(creator=1).filter(offer_version=2).exists()
+    request = get_request(2)
+    VoteFormView.as_view()(request)
+    assert Vote.objects.filter(creator=1).filter(offer_version=2).get()
+    assert not Vote.objects.filter(creator=1).filter(offer_version=2)[0].approval
```

Comment:
```suggestion
    vote = Vote.objects.get(creator=1, offer_version=2)
    assert not vote.approval
```
- `filter().filter().get()` does not really make sense - just use `get()` in the first place.
- `filter()[0]` does not make sense - either use `get()` if you know exactly one element exists or `filter().first()` if you're unsure how many objects might exist with these filters.